examples/load_and_reindex_bboxes.py

```diff
@@ -0,0 +1,336 @@
+# in the missing frames, by plotting the bounding boxes
+# for the first 6 frames of the video.
+#
+# Remember only frames 0 and 5 are annotated in the original dataset. These
+# are plotted in blue, while the forward filled values are plotted in orange
+# and the linearly interpolated values in green.
+
+# sphinx_gallery_thumbnail_number = 2
+
+# initialise figure
+fig = plt.figure(figsize=(8, 8))
+
+# loop over frames
+for frame_n in range(6):
+    # add subplot axes
+    ax = plt.subplot(3, 2, frame_n + 1)
+
+    # plot frame
+    # note: the video is indexed at every frame, so
+    # we use the frame number as index
+    ax.imshow(video[frame_n])
+
+    # plot bounding box for each dataset
+    for ds_i, ds_one in enumerate([ds_nan, ds_ff, ds_interp]):
+        # plot box
+        top_left_corner = (
+            ds_one.position.sel(time=frame_n, individuals="id_1").data
+            - ds_one.shape.sel(time=frame_n, individuals="id_1").data / 2
+        )
+        bbox = plt.Rectangle(
+            xy=tuple(top_left_corner),
+            width=ds_one.shape.sel(
+                time=frame_n, individuals="id_1", space="x"
+            ).data,
+            height=ds_one.shape.sel(
+                time=frame_n, individuals="id_1", space="y"
+            ).data,
+            edgecolor=list_colors[ds_i],
+            facecolor="none",
+            # make line for NaN dataset thicker and dotted
+            linewidth=[5, 1.5, 1.5][ds_i],
+            linestyle=["dotted", "solid", "solid"][ds_i],
+            label=["nan", "ffill", "linear"][ds_i],
+        )
+        ax.add_patch(bbox)
+
+        # plot centroid
+        ax.scatter(
+            x=ds_one.position.sel(
+                time=frame_n, individuals="id_1", space="x"
+            ).data,
+            y=ds_one.position.sel(
+                time=frame_n, individuals="id_1", space="y"
+            ).data,
+            s=5,
+            color=list_colors[ds_i],
+        )
+
+    # add legend to first frame
+    if frame_n == 0:
+        ax.legend()
+    ax.set_title(f"Frame {frame_n}")
+    ax.set_xlabel("x (pixels)")
+    ax.set_ylabel("y (pixels)")
+
+fig.tight_layout()
+
+# %%
+# Export as .csv file
+# -------------------
+# Let's assume the dataset with the forward filled values is the best suited
+# for our task - we can now export the computed values to a .csv file
+#
+# Note that we currently do not provide explicit methods to export a
+# ``movement`` bounding boxes dataset in a specific format. However, we can
+# easily save the bounding boxes’ trajectories to a .csv file using the
+# standard Python library ``csv``.
+
+# define name for output csv file
+filepath = "tracking_output.csv"
+
+# open the csv file in write mode
+with open(filepath, mode="w", newline="") as file:
+    writer = csv.writer(file)
+
+    # write the header
+    writer.writerow(
+        ["frame_idx", "bbox_ID", "x", "y", "width", "height", "confidence"]
```

Comment:
Do we know the fps for this data? If so, we could even add a 'time' column?
Or perhaps this is scope creep and just ignore me.

Reply:
I think it's probably nice to keep it simple in this case and stick to frames